src/commonMain/resources/db/migration/V70__FollowsCollabs.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE public.follows
+    ADD COLUMN "collab" boolean NOT NULL DEFAULT TRUE;
```

Comment:
Just for my sanity we should probably run an initial setup like
```sql
UPDATE public.follows SET "collab" = "upload";
```

So that people who are not following someone anymore aren't somehow set to collab. I know the query for alerts filters it out but there could be some weird side effects when changing follower preferences...